Exam Labs/2013-12-20 Cloth Cap/lab/lab.conf

```diff
@@ -0,0 +1,33 @@
+as20r1[0]=L
```

Comment:
Please, add at the start of the lab.conf the metadata (you can see an example other labs) and feel free to put your name as author :)